app/src/main/java/com/bytemedrive/settings/root/EventSettingsThumbnailCreationDisabled.kt

```diff
@@ -0,0 +1,27 @@
+package com.bytemedrive.settings.root
+
+import android.util.Log
+import com.bytemedrive.application.encryptedSharedPreferences
+import com.bytemedrive.database.ByteMeDatabase
+import com.bytemedrive.store.Convertable
+
+data class EventSettingsThumbnailCreationDisabled(
```

Comment:
why 2 events for same thing? Keep only this one. Rename it to `EventThumbnailCreationSet`